components/lib/crash/src/test/java/mozilla/components/lib/crash/CrashReporterTest.kt

```diff
@@ -198,7 +199,7 @@ class CrashReporterTest {
         reporter.onCrash(
             mock(),
             Crash.UncaughtExceptionCrash(RuntimeException(), arrayListOf()))
-
+        sleep(100)
```

Comment:
Could we use `runBlocking` as `sleep(N) `could make the test unreliable?

Reply:
Agreed. I'll look into another way to do this better.  Thanks,